testdata/jennies/builders/package-with-dashes/PythonBuilder/builders/builder-pkg.py

```diff
@@ -0,0 +1,19 @@
+import typing
+from ..cog import builder as cogbuilder
+from ..models import with-dashes
+
+
+class SomeNiceBuilder(cogbuilder.Builder[with-dashes.SomeStruct]):    
```

Comment:
The pythonic equivalent would be an underscore. I think what would work best for python is the same logic as golang but instead of replacing by an empty space, replace with an underscore

```suggestion
class SomeNiceBuilder(cogbuilder.Builder[with_dashes.SomeStruct]):    
```